src/components/navigation-page/navigation-page.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+export default class NavigationPage extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: null,
+      activeParent: null
+    };
+  }
+
+  componentDidMount() {
+    const { location, parentPage } = this.props;
+
+    this.setState({
+      activeItem: location.pathname,
+      activeParent: parentPage
+    });
```

Comment:
Since this component isn't interactive, I don't think we need to use state. Instead we should be able to set the component based on the props passed.